monitoring/jaeger-mixin/README.md

```diff
@@ -0,0 +1,82 @@
+# Prometheus monitoring mixin for Jaeger
+
+The Prometheus monitoring mixin for Jaeger provides a starting point for people wanting to monitor Jaeger using Prometheus, Alertmanager, and Grafana. To use it, you'll need [`jsonnet`](https://github.com/google/go-jsonnet) and [`jb` (jsonnet-bundler)](https://github.com/jsonnet-bundler/jsonnet-bundler). They can be installed using `go get`, as follows:
+
+```console
+$ go get github.com/google/go-jsonnet/cmd/jsonnet
+$ go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+```
+
+Your monitoring mixin can then be initialized as follows:
+
+```console
+$ jb init
+$ jb install \
+  github.com/jaegertracing/jaeger/monitoring/jaeger-mixin@master \
+  github.com/grafana/jsonnet-libs/grafana-builder@master \
+  github.com/coreos/kube-prometheus/jsonnet/kube-prometheus@master
+```
+
+In the directory your mixin was initialized, create a new `monitoring-setup.jsonnet` file like the following. Note that this file is yours, any customizations to Prometheus, Grafana, or Alertmanager should take place here.
+
+```jsonnet
+local jaegerAlerts = (import 'jaeger-mixin/alerts.libsonnet').prometheusAlerts;
+local jaegerDashboard = (import 'jaeger-mixin/mixin.libsonnet').grafanaDashboards;
+
+local kp =
+  (import 'kube-prometheus/kube-prometheus.libsonnet') +
+  {
+    _config+:: {
+      namespace: 'monitoring',
+    },
+    grafanaDashboards+:: {
+      'jaeger.json': jaegerDashboard['jaeger.json'],
+    },
+    prometheusAlerts+:: jaegerAlerts,
+  };
+
+{ ['00namespace-' + name + '.json']: kp.kubePrometheus[name] for name in std.objectFields(kp.kubePrometheus) } +
```

Comment:
It looks like in the later files string interpolation is used. Is it not possible here, eg |||00namespace-{{ $name }}.json||| ?

Would make it easier to read.

Reply:
I'll see if that's supported and add to the follow-up PR that I'll send with the comments for each property.

Reply:
https://github.com/jaegertracing/jaeger/issues/1714